Subject: Adopt to Garden Linux defaults

--- org/debian/rules
+++ new/debian/rules
@@ -58,16 +58,16 @@
 	-Ddefault-kill-user-processes=false \
 	-Ddefault-hierarchy=unified \
 	-Ddefault-dnssec=no \
-	-Dpamconfdir=no \
+	-Dpamconfdir=/etc/pam.d \
 	-Dpamlibdir=/lib/$(DEB_HOST_MULTIARCH)/security \
 	-Drpmmacrosdir=no \
-	-Dqrencode=false \
+	-Dqrencode=true \
 	-Dvconsole=false \
 	-Dfirstboot=true \
 	-Dsysupdate=false \
 	-Dxkbcommon=false \
 	-Dwheel-group=false \
-	-Dpwquality=false \
+	-Dpwquality=true \
 	-Dp11kit=false \
 	-Doomd=true \
 	-Durlify=true \
--- org/debian/control
+++ org/debian/control
@@ -67,6 +67,8 @@
                zstd <!nocheck>,
                gawk <!nocheck>,
                fdisk <!nocheck>,
+               libpwquality-dev,
+               libqrencode-dev
 
 Package: systemd
 Architecture: linux-any
