Subject: Adopt to Garden Linux defaults

+++ a/debian/rules
@@ -58,7 +58,7 @@
 	-Ddefault-kill-user-processes=false \
 	-Ddefault-hierarchy=unified \
 	-Ddefault-dnssec=no \
-	-Dpamconfdir=no \
+	-Dpamconfdir=/etc/pam.d \
 	-Dpamlibdir=/lib/$(DEB_HOST_MULTIARCH)/security \
 	-Drpmmacrosdir=no \
 	-Dvconsole=false \
@@ -66,7 +66,7 @@
 	-Dsysupdate=false \
 	-Dxkbcommon=false \
 	-Dwheel-group=false \
-	-Dpwquality=false \
+	-Dpwquality=true \
 	-Dp11kit=false \
 	-Doomd=true \
 	-Durlify=true \
@@ -138,7 +138,7 @@
 	-Dopenssl=false \
 	-Dlibfido2=false \
 	-Dtpm2=false \
-	-Dqrencode=false \
+	-Dqrencode=true \
 	-Dhomed=false \
 	-Duserdb=false \
 	-Dpcre2=false
+++ a/debian/control
@@ -70,6 +70,8 @@
                zstd <!nocheck>,
                gawk <!nocheck>,
                fdisk <!nocheck>,
+               libpwquality-dev,
+               libqrencode-dev
 
 Package: systemd
 Architecture: linux-any
