diff --color -Naur a/debian/rules b/debian/rules
--- a/debian/rules	2024-05-07 12:37:08.191863463 +0200
+++ b/debian/rules	2024-05-07 12:40:02.399945660 +0200
@@ -62,7 +62,7 @@
 	-Dsupport-url=$(SUPPORT_URL) \
 	-Ddefault-kill-user-processes=false \
 	-Ddefault-dnssec=no \
-	-Dpamconfdir=no \
+	-Dpamconfdir=/etc/pam.d \
 	-Dpamlibdir=/usr/lib/$(DEB_HOST_MULTIARCH)/security \
 	-Drpmmacrosdir=no \
 	-Dvconsole=false \
@@ -153,7 +153,7 @@
 	-Dtpm2=disabled \
 	-Dp11kit=disabled \
 	-Dqrencode=disabled \
-	-Dpwquality=disabled \
+	-Dpwquality=true \
 	-Dhomed=disabled \
 	-Duserdb=false \
 	-Dpcre2=disabled \
