Subject: Adopt to Garden Linux defaults

--- org/debian/rules	2021-10-17 21:36:55.100237147 +0000
+++ new/debian/rules	2021-10-17 21:38:35.618910510 +0000
@@ -57,17 +57,17 @@ CONFFLAGS = \
 	-Ddefault-kill-user-processes=false \
 	-Ddefault-hierarchy=unified \
 	-Ddefault-dnssec=no \
-	-Dpamconfdir=no \
+	-Dpamconfdir=/etc/pam.d \
 	-Dpamlibdir=/lib/$(DEB_HOST_MULTIARCH)/security \
 	-Drpmmacrosdir=no \
-	-Dqrencode=false \
+	-Dqrencode=true \
 	-Dvconsole=false \
 	-Dfirstboot=false \
 	-Dxkbcommon=false \
 	-Dwheel-group=false \
-	-Duserdb=false \
+	-Duserdb=true \
 	-Dhomed=false \
-	-Dpwquality=false \
+	-Dpwquality=true \
 	-Dp11kit=false \
 	-Doomd=true \
 	-Durlify=false \
--- org/debian/control	2021-10-17 21:40:22.989765859 +0000
+++ new/debian/control	2021-10-17 21:41:45.743966420 +0000
@@ -65,6 +65,8 @@ Build-Depends: debhelper-compat (= 13),
                zstd <!nocheck>,
                gawk <!nocheck>,
                fdisk <!nocheck>,
+               libpwquality-dev,
+               libqrencode-dev,

 Package: systemd
 Architecture: linux-any
